networkx/algorithms/bidirectional.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
+#    Copyright (C) 2015 by
+#    André Dietrich <dietrich@ivs.cs.uni-magdeburg.de>
+#    All rights reserved.
+#    BSD license.
+import networkx as nx
+__author__ = """\n""".join(['André Dietrich <dietrich@ivs.cs.uni-magdeburg.de>',
+                            'Sebastian Zug <zug@ivs.cs.uni-magdeburg.de>'])
```

Comment:
What about keeping this variable only in the comment above? `__author__` is not being used now-a-days.